src/control/server/config_types.go

```diff
@@ -237,9 +237,12 @@ type configuration struct {
 	NrHugepages    int             `yaml:"nr_hugepages"`
 	ControlLogMask ControlLogLevel `yaml:"control_log_mask"`
 	ControlLogFile string          `yaml:"control_log_file"`
+	Targets        int             `yaml:"targets"`
 	// development (subject to change) config fields
 	Modules   string
 	Attach    string
+	XShelpernr int
```

Comment:
these will not be parsed in the config file, as they don't have a yaml tag mapping them to a config parameter, is this intentional?

Reply:
not intentional, please feel free to refine it at your convenient time, thanks.

"so -t will take preference if both -t & -c are specified?"
yes, the "-c" is just to keep compatible with old usage (will with a msg printed that that options is deprecated).

"currently -c can take a range of cores (although it is just converted to a count for the moment), is that something we want users to be able to specify or not (Johann already mentioned this in comments)?"
I think yes, per Johann's comment above.